OSDOCS 10788 :Updated oc-mirror v2 docs for 4.18 release. (#88188)
openshift/openshift-docs: modules/oc-mirror-dry-run-v2.adoc

+
[source,terminal]
----
$ oc mirror -c <image_set_config_yaml> file://<oc_mirror_workspace_path> --dry-run --v2
----
+

> Where:

Reviewer: [Docs guidelines section on how to format description lists](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#:~:text=of%20requested%20CPU.-,If%20you%20must,-provide%20additional%20information), I'm going to make several comments to these lines based on that section
```suggestion
where:
```

> - `<image_set_config_yaml>`: Use the image set configuration file that you just created.

Reviewer: Remove the hyphens from these items since they don't need to be a bullet list. Also, add another colon after the replaced value:
```suggestion
`<image_set_config_yaml>`:: Use the image set configuration file that you just created.
```

> - `<image_set_config_yaml>`: Use the image set configuration file that you just created.

Reviewer: Also, the descriptions should be written similar to the docs guidelines where the sentence starts with "Specifies...":
```suggestion
- `<image_set_config_yaml>`: Specifies the image set configuration file that you created.
```

> 

Reviewer: I think these blank lines in between the values can be deleted:
```suggestion
```

- `<oc_mirror_workspace_path>`: Insert the address of the workspace path.

- `<mirror_registry_url>`: Insert the URL or address of the remote container registry from which images will be mirrored or deleted.
+
.Example output
[source,terminal]
----

> $ oc mirror --config /tmp/isc_dryrun.yaml file://<oc_mirror_workspace_path> --dry-run --v2
> 

Reviewer: Didn't realize this before but if this is example output, the actually command needs to be removed from the source block. Example output blocks should only have the actual output:
```suggestion
```
